CMakeLists.txt

```diff
@@ -1,36 +1,61 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
 
 project(Kvasir)
 
 enable_testing()
 
+#Enable C++11 per default on all targets
+set(CMAKE_CXX_STANDARD 11)
+set(CXX_STANDARD_REQUIRED TRUE)
+
 set(CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-  # using Clang
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-  # using GCC
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -pedantic -Wold-style-cast -I ${CMAKE_CURRENT_SOURCE_DIR}/Lib/")
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
```

Comment:
intel and MSVC are actually useless here, they have no cortex support, this shows how lazy I was setting this up. They can be removed as well